buildbot/master/files/config/master.cfg

```diff
@@ -157,10 +157,7 @@ c['builders'] = [
     DynamicServoBuilder("arm64", CROSS_SLAVES, envs.build_arm64),
     DynamicServoBuilder("linux-dev", LINUX_SLAVES, envs.build_linux),
     DynamicServoBuilder("linux-nightly", LINUX_SLAVES, envs.build_linux),
-    DynamicServoBuilder("linux-rel-css", LINUX_SLAVES,
-                        envs.build_linux_rel_debug_assert),
-    DynamicServoBuilder("linux-rel-wpt", LINUX_SLAVES,
-                        envs.build_linux_rel_debug_assert),
+    DynamicServoBuilder("linux-rel", LINUX_SLAVES, envs.build_linux),
```

Comment:
We don't have this builder right now. The change should be such that `linux-rel-css` and `linux-rel-wpt` take envs.build_linux for their environments.